docs/criufeature.md

```diff
@@ -0,0 +1,93 @@
+
+For more information about the CRIU feature and OpenJ9 CRIU support, see the following blog posts:
+
+- [Fast JVM startup with OpenJ9 CRIU Support](https://blog.openj9.org/2022/09/26/fast-jvm-startup-with-openj9-criu-support/)
+- [Getting started with OpenJ9 CRIU Support](https://blog.openj9.org/2022/09/26/getting-started-with-openj9-criu-support/)
+- [OpenJ9 CRIU Support: A look under the hood](https://blog.openj9.org/2022/10/14/openj9-criu-support-a-look-under-the-hood/)
+- [OpenJ9 CRIU Support: A look under the hood (part II)](https://blog.openj9.org/2022/10/14/openj9-criu-support-a-look-under-the-hood-part-ii/)
+
+## Using the CRIU feature
+
+CRIU feature is not enabled by default, you must explicitly invoke it. You can invoke the feature with the `-XX:+EnableCRIUSupport` option. When thus enabled, the VM starts with support for checkpoint and restore. The VM will launch the JIT, GC, Security, and other internal components that make checkpoint and restore feasible.
+
+For more information about the `-XX:+EnableCRIUSupport` option, see the [`-XX:[+|-]EnableCRIUSupport`](xxenablecriusupport.md) topic.
+
+## Configuring the CRIU support feature
+
+You can use command line options to configure the CRIU support to use CRIU capabilities that are provided by the operating system as well as VM support.
+
+### New options for checkpoint and restore process
+
+- [`-XX:[+|-]EnableCRIUSupport`](xxenablecriusupport.md) - Enables starting of the VM with support for checkpoint and restore.
+- [`-XX:[+|-]CRIURestoreNonPortableMode`](xxcriurestorenonportablemode.md) - Indicates to the VM that only a single checkpoint can be taken in the VM lifetime.
+- [`-XX:CheckpointGCThreads`](xxcheckpointgcthread.md) - Specifies the number of GC threads that should be used at the checkpoint time.
+- [`Dorg.eclipse.openj9.criu.ImmutableEnvVars=[ENV_VAR1,ENV_VAR2,...ENV_VARN]`](dorgeclipseopenj9criuimmutableenvvars.md) - Specifies the environment variables that should not be allowed to change through `CRIUSupport.registerRestoreEnvFile` upon restore.
+
+### New options at restore only
+
+- [`-XX:[+|-]IgnoreUnrecognizedRestoreOptions`](xxignoreunrecognizedrestoreoptions.md) - Disables the VM failure and allows the VM to continue even if some VM options that are restored are not supported or do not exist.
+- [`-Xshareclasses:disableOnRestore`](xshareclassesdisableonrestore.md) - Disables SCC on restore.
```

Comment:
SCC needs expanding and should have a "the" - "the shared classes cache". And it should probably be "Disables further use of the shared classes cache" since the issue says that the SCC API will still be available.